src/Microsoft.AspNetCore.Mvc.Razor.Host/Internal/ViewComponentTagHelperChunkVisitor.cs

```diff
@@ -82,7 +88,7 @@ private void WriteClass(TagHelperDescriptor descriptor)
             {
                 // Add view component helper.
                 Writer.WriteVariableDeclaration(
-                    $"private readonly {_context.IViewComponentHelperTypeName}",
+                    $"private readonly global::{_context.IViewComponentHelperTypeName}",
```

Comment:
Should the `global::` be part of the property definition? What do we do in other places @NTaylorMullen